eisbuk-admin/.babelrc.js

```diff
@@ -1,4 +1,5 @@
 const plugins = [
+  ["@babel/plugin-proposal-private-methods", { loose: true }],
```

Comment:
Thanks! These warnings were about to drive me crazy!

Reply:
:wink: did it just now :sunglasses: